index.js

```diff
@@ -59,14 +62,17 @@ var pluralTypes = {
   icelandic: function (n) { return (n % 10 !== 1 || n % 100 === 11) ? 1 : 0; }
 };
 
-// Mapping from pluralization group to individual locales.
+// Mapping from pluralization group to individual language codes/locales.
+// Will look up based on exact match, if not found and it's a locale will parse the locale
+// for language code, and if that does not exist will default to 'en'
 var pluralTypeToLanguages = {
   arabic: ['ar'],
-  chinese: ['id', 'ja', 'ko', 'lo', 'ms', 'th', 'tr', 'zh'],
-  german: ['fa', 'da', 'de', 'en', 'es', 'fi', 'el', 'he', 'hu', 'it', 'nl', 'no', 'pt', 'sv'],
+  chinese: ['id', 'ja', 'ko', 'ko-KR', 'lo', 'ms', 'th', 'th-TH', 'tr', 'zh'],
```

Comment:
i don't think we need to add the exact locales here, when they're covered by the language code.

Reply:
Isn't it better to be more specific then general.  For two reason:
1) It removes confusion?  For example, when i looked at it, I was confused but with the addition of the 
2) In the case when the locale is actually different then the language code.  For example, not sure if in the script languages will the pural rules change between locales (cyrillic vs latin)

It does no harm to include it, but it may not be true the other way.

Reply:
I'll leave it as then if the first point is acceptable?  Let me know if there is any other issue that i need to address before you can accept the PR.

Thanks!!

Reply:
sure, that seems fine. i'll review and merge this tomorrow (it'd be great to split up the commit into two: one adding more specific locales, and another solely fixing croatian - but i can do that for you prior to merging, so you're welcome to let it stand as-is)

Reply:
Another interesting thing is that it's a little confusing how it's grouped.  Since it's grouping by different plural forms, but using a specific country as the variable names.  Example you incorporate korean into the Chinese group.  That's a big confusing.